### DISCUSSION TITLE
Force-reloading TileLayer tiles?

### DISCUSSION THREAD

**Discussion by j6k4m8** [visgl/deck.gl#5312] [Q&A] [answered]
Is there an easy way to force a reload of all tiles in a `TileLayer`? I have data that change over time and I want to be able to automatically reload the tiles (source URL and other params are all the same). I was hoping to avoid having several TileLayers and throwing them away and creating new ones as I need them.

I think I just need a "cache-flush" function call somewhere (?)

**Comment by Pessimistress** [comment, marked as answer]
Use [updateTriggers](https://deck.gl/docs/api-reference/core/layer#updatetriggers) to invalidate `getTileData`.

**Reply by j6k4m8** [reply]
Exactly what I needed — thank you a million!
